Formula/t/tfenv.rb

```diff
@@ -29,6 +29,7 @@ class Tfenv < Formula
 
   def install
     prefix.install %w[bin lib libexec share]
+    bin.env_script_all_files prefix/"rbin", TFENV_CONFIG_DIR: etc/"tfenv"
```

Comment:
`rbin` seems like an odd choice here.

Let's use `HOME` to be nicer for multi-user environments, and avoid clobbering existing `TFENV_CONFIG_DIR` configuration:

```suggestion
    bin.env_script_all_files prefix/"rbin", TFENV_CONFIG_DIR: "${TFENV_CONFIG_DIR:-${XDG_CONFIG_HOME:-${HOME}/.config}/tfenv}"
```
May need extra quoting, I forget how `env_script_all_files` handles this.

Reply:
We normally use `libexec`, but presumably we avoid that here since `tfenv` already makes use of `libexec`. What do we risk if we use `libexec`?

Or, maybe, `share`? FHS describes it as "architecture-independent data", which seems appropriate.

That said, not hugely opposed to `prefix/"realbin"`, so we can go with that if you think that's best.

Reply:
> We normally use `libexec`, but presumably we avoid that here since `tfenv` already makes use of `libexec`. What do we risk if we use `libexec`?

If upstream later decides to do the same thing (move the core `tfenv` to `libexec` and put a wrapper script in its place), major breakage results that may be hard to trace. (I just tested creating a `libexec/tfenv` dummy file, and `env_script_all_files` silently overwrote it with the core `tfenv` script.) 

> Or, maybe, `share`? FHS describes it as "architecture-independent data", which seems appropriate.

This also breaks the formula if upstream decides to populate a `share/tfenv` directory, because `env_script_all_files` will silently move the source script inside the directory without warning, which makes the upstream `tfenv` script unable to find the `libexec` subcommands.

I'm basically trying to avoid silently doing the wrong thing. 😁

Reply:
OK, I'll switch to `libexec`, since there aren't any conflicts for this version, and I've initiated https://github.com/Homebrew/brew/pull/20281 to catch future conflicts.